Tests/Tests_REST_API/test_base_page.py

```diff
@@ -0,0 +1,31 @@
+from Tests.Tests_REST_API.BaseClass import ParaBankHomePage
+from Tests.Tests_REST_API.TestData import DataEndPoints
+
+
+class TestParaBank(ParaBankHomePage):
+
+    def test_get_account_by_id(self):
+        response = self.get(f"{DataEndPoints.base_uri}{DataEndPoints.getAccountId}")
```

Comment:
please write this like DataEndPoints.base_uri + DataEndPoints.getAccountId
it is more readable